expediaRequester/requester.py

```diff
@@ -74,26 +74,19 @@ def hotel_reviews(self, hotelId, summary, sortBy, start, items, categoryFilter):
     Start cars API
     """
 
-    def cars_search(self, pickupDate, dropoffDate, pickupLocation, dropoffLocation, sort, limit, suppliers, classes):
-        url = "http://terminal2.expedia.com:80/x/cars/search?"
-        if pickupDate is not None:
-            url += "pickupdate=" + pickupDate
-        if dropoffDate is not None:
-            url += "&dropoffdate=" + dropoffDate
-        if pickupLocation is not None:
-            url += "&pickuplocation=" + pickupLocation
-        if dropoffLocation is not None:
-            url += "&dropofflocation=" + dropoffLocation
-        if sort is not None:
-            url += "&sort=" + sort
-        if limit is not None:
-            url += "&limit=" + str(limit)
-        if suppliers is not None:
-            url += "&suppliers=" + suppliers
-        if classes is not None:
-            url += "&classes=" + classes
-        url = url + "&apikey=" + self.apiKey
-        return self.get(url)
+    def cars_search(self, pickupDate = None, dropoffDate = None, pickupLocation = None, dropoffLocation = None, sort = None, limit = None, suppliers = None, classes = None):
+        payload = {}
+        url = "http://terminal2.expedia.com:80/x/cars/search"
+        payload['pickupdate'] = pickupdate
+        payload['dropoffdate'] = dropoffDate
+        payload['pickuplocation'] = pickuplocation
+        payload['dropofflocation'] = dropofflocation
```

Comment:
Similarly pickupLocation and dropoffLocation

I know these are not the python conventions but I'll fix them in the next commit. For now just stick to dropoffLocation and pickupLocation for the test cases to pass.